### PR TITLE
Storage follows the duck type for attribute overrides.

### PR DESCRIPTION
**Pull request by Harwayne**
Storage follows the duck type for attribute overrides.

### CODE REVIEW

File: pkg/reconciler/storage/storage.go [hunk 1]
```diff
@@ -280,8 +280,10 @@ func (c *Reconciler) reconcileNotification(ctx context.Context, storage *v1alpha
 	}
 
 	customAttributes := make(map[string]string)
-	for k, v := range storage.Spec.CustomAttributes {
-		customAttributes[k] = v
+	if storage.Spec.CloudEventOverrides != nil {
```
Comment (n3wscott) [inline pkg/reconciler/storage/storage.go]: the cloudEventOverrides should propagate down into the PullSubscription, not added to the customAttributes

**Comment by n3wscott** [comment]
/hold

Leverage the PullSubscription CloudEventsOverrides

File: pkg/reconciler/storage/storage.go (hunk 1 above)
Comment (Harwayne) [reply pkg/reconciler/storage/storage.go]: I don't understand the tradeoffs, so I made the minimum change to match the desired interface.

**Comment by Harwayne** [comment]
> /hold
> 
> Leverage the PullSubscription CloudEventsOverrides

I don't understand the tradeoffs, so I made the minimum change to match the desired interface.

/hold cancel

**Comment by n3wscott** [comment]
/lgtm

I made an issue to follow up

**Pull request merged**